common/src/test/java/com/microsoft/identity/common/BrokerOAuth2TokenCacheTest.java

```diff
@@ -284,20 +287,23 @@ private void initOtherCaches(final Context context) {
         return fileManagers;
     }
 
-    private ISharedPreferencesFileManager getAppUidFileManager(final Context context,
+    private ISharedPreferencesFileManager getAppUidFileManager(final ICommonComponents components,
                                                                final int appUid) {
-        return SharedPreferencesFileManager.getSharedPreferences(
-                context,
+        if (!(components instanceof AndroidCommonComponents)) {
+            throw new IllegalStateException("This component must be migrated to support the new platform abstraction");
+        }
+        return components.getEncryptedFileStore(
                 getBrokerUidSequesteredFilename(appUid),
-                new AndroidAuthSdkStorageEncryptionManager(context, null)
-        );
+                new AndroidAuthSdkStorageEncryptionManager(((AndroidCommonComponents) components).getPlatformContext(), null));
```

Comment:
re: platformContext

In this case, we should be getting AndroidAuthSdkStorageEncryptionManager from ICommonComponent instead.

Reply:
Sure - but that method is currently Performing Magic(tm).  I could expose a specific one to avoid it.